backend/api/models.py

```diff
@@ -13,18 +17,22 @@ class CoursePlan(models.Model):
     startAt = models.DateField()
     endAt = models.DateField()
     # even week: True, odd week: False
-    WEEKTYPE = [(True, 'EVEN'), (False, 'ODD')]
     firstWeekType = models.BooleanField(choices=WEEKTYPE)
 
+    def __str__(self):
+        return '{}, {}, {}, {}'.format(self.faculty, self.course, self.studiesMode, self.term)
+
 
-# table of users
-class User(models.Model):
-    userID = models.AutoField(primary_key=True)
-    login = models.CharField(max_length=30)
-    password = models.CharField(max_length=256)  # length due to encryption
+# table of course plans users
+class PlanUser(models.Model):
+    username = User.username
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='db_user')
     group = models.ForeignKey('StudentGroup', on_delete=models.SET_NULL, null=True)
     coursePlan = models.ForeignKey('CoursePlan', on_delete=models.SET_NULL, null=True)
 
+    def __str__(self):
+         return 'Username: {} groupID: {} coursePlanID: {}'.format(self.username, self.group_id, self.coursePlan)
```

Comment:
The same:
return f"Username: {self.username} groupID: {self.group_id} coursePlanID: {self.coursePlan}"